code/game/objects/items/weapons/tools.dm

```diff
@@ -34,6 +34,13 @@
 	playsound(loc, 'sound/weapons/genhit.ogg', 50, 1, -1)
 	return (BRUTELOSS)
 
+/obj/item/weapon/wrench/golden
+	name = "golden wrench"
+	desc = "Destroying one of these makes fat neckbeards around the galaxy cry."

```

Comment:
I'm glad we got this blazing team fortress 2 joke in while it was still fresh and relevant to the collective consciousness of _gamer culture_